packages/portal/src/components/item/ItemLanguageSelector.vue

```diff
@@ -19,6 +19,21 @@
                 class="pr-1"
                 data-qa="translate item error"
               />
+              <i18n
+                v-else-if="!$auth.loggedIn"
+                path="multilingual.loginToTranslate"
+                tag="span"
+                data-qa="translate item login suggestion"
+              >
+                <template #login>
+                  <b-link
+                    data-qa="log in button"
```

Comment:
This "link" can't be opened in a new tab. Make it a real link like the main log in (in top nav).

Reply:
Done.